jetty-core/jetty-fcgi/jetty-fcgi-client/src/main/java/org/eclipse/jetty/fcgi/generator/ServerGenerator.java

```diff
@@ -83,50 +83,48 @@ public Result generateResponseHeaders(int request, int code, String reason, Http
         // End of headers
         length += EOL.length;
 
-        ByteBuffer buffer = acquire(length);
-        BufferUtil.clearToFill(buffer);
+        ByteBuffer byteBuffer = BufferUtil.allocate(length, isUseDirectByteBuffers());
```

Comment:
Because in FastCGI the headers are sent back to the client as if they were content.
This method later calls `generateContent()` which is also used elsewhere for real content (not headers), so this call needed an extra `boolean` parameter to say whether the "content" must be released or not, and this was the only place where this was true.

Rather than carrying an extra parameter, I decided to just allocate on-the-fly as if it was any other content for simplicity.
Also, being server-side FastCGI, not worried about performance.